scripts/context_combiner.rb

```diff
@@ -0,0 +1,21 @@
+require 'JSON'
```

Comment:
Is it useful to keep this script in here? Is it largely for development or am I missing a call somewhere?

Reply:
It's just for development. I left it because I thought as data changed over time someone might want to run it again someday. but I'm not attached to it!

Reply:
Can you add a comment to the top about why one might run it?